src/adminPage/adminPage.component.tsx

```diff
@@ -0,0 +1,232 @@
+import React, { useState } from 'react';
+import Typography from '@material-ui/core/Typography';
+import {
+  Theme,
+  StyleRules,
+  createStyles,
+  WithStyles,
+  withStyles,
+  Paper,
+  TextareaAutosize,
+  FormControlLabel,
+  Checkbox,
+  Button,
+} from '@material-ui/core';
+import { connect } from 'react-redux';
+import { StateType } from '../state/state.types';
+import {
+  AppStrings,
+  MaintenanceState,
+  ScheduledMaintenanceState,
+} from '../state/scigateway.types';
+import { getString, getAppStrings } from '../state/strings';
+import { ThunkDispatch } from 'redux-thunk';
+import { AnyAction } from 'redux';
+import {
+  setMaintenanceState,
+  setScheduledMaintenanceState,
+} from '../state/actions/scigateway.actions';
+
+const styles = (theme: Theme): StyleRules =>
+  createStyles({
+    root: {
+      padding: theme.spacing(2),
+    },
+    titleText: {
+      color: theme.palette.secondary.main,
+      fontWeight: 'bold',
+    },
+    paper: {
+      display: 'flex',
+      flexDirection: 'column',
+      marginTop: theme.spacing(2),
+      marginBottom: theme.spacing(2),
+      padding: theme.spacing(2),
+      [theme.breakpoints.up(800 + theme.spacing(4))]: {
+        width: 800,
+        marginLeft: 'auto',
+        marginRight: 'auto',
+      },
+    },
+    form: {
+      flexDirection: 'column',
+    },
+    textArea: {
+      backgroundColor: 'inherit',
+      color: theme.palette.text.primary,
+      font: 'inherit',
+      marginTop: theme.spacing(1),
+      marginBottom: theme.spacing(1),
+      minWidth: '40%',
+      maxWidth: '100%',
+    },
+  });
+
+interface AdminPageProps {
+  scheduledMaintenance: ScheduledMaintenanceState;
+  maintenance: MaintenanceState;
+  res: AppStrings | undefined;
+}
+
+interface AdminPageDispatchProps {
+  setScheduledMaintenanceState: (
+    scheduledMaintenanceState: ScheduledMaintenanceState
+  ) => Promise<void>;
+  setMaintenanceState: (maintenanceState: MaintenanceState) => Promise<void>;
+}
+
+export type CombinedAdminPageProps = AdminPageProps &
+  AdminPageDispatchProps &
+  WithStyles<typeof styles>;
+
+const AdminPage = (props: CombinedAdminPageProps): React.ReactElement => {
+  const [scheduledMaintenance, setScheduledMaintenance] = useState<
+    ScheduledMaintenanceState
+  >(props.scheduledMaintenance);
+  const [maintenance, setMaintenance] = useState<MaintenanceState>(
+    props.maintenance
+  );
+
+  return (
+    <div className={props.classes.root}>
+      <Typography variant="h3" className={props.classes.titleText}>
+        {getString(props.res, 'title')}
```

Comment:
I don't think it's necessarily a problem, but on wide windows the title being left aligned and the actual content being centred looks a bit strange to me. I know the title is left aligned on the cookies page, but on the login page (which also uses centred content) we don't have a left aligned `h3` header at the top of the page, just the `h5` header centred within the paper. I wonder if that might look better on the admin page?

![Screen Shot 2021-03-01 at 17 10 59](https://user-images.githubusercontent.com/61705287/109532590-2bf1d080-7ab1-11eb-8f97-d840a0bf07cc.png)

Reply:
I kept it left aligned so that it's consistent with the _Contact_, _Help_ and _Cookies_ pages. I will be adding new content to the Admin page in another issue so I will decide what's best to do with the title after making the changes in that issue.